### DISCUSSION TITLE
Ramen recipe

### DISCUSSION THREAD

**Discussion by LEWeee** [Lgmrszd/TFC-TNG_BWM_GT-CE#1] [Ideas]
I created this issue for 2 reasons:
1.) It's one of the first issues we got regarding this modpack
2.) I though it would be funny if the first issue was about this

The problem is that the recipe for Ramen Dough in Sakura requires the use of Alkali.
![image](https://user-images.githubusercontent.com/16687385/101986246-34462680-3c8d-11eb-926c-9db13d0693ee.png)

In Sakura, Alkali is by default made in the Stone Mortar out of Stone, though this in bit of a dispute with TFC's design philosophy. While it's not exactly known why is Alkali made this way I assume it's that way to be consistent with Sakura's table Salt recipe (Alkali recipe - top, Salt recipe - bottom)
![image](https://user-images.githubusercontent.com/16687385/101986448-2ba22000-3c8e-11eb-9948-8b8cd093c75c.png)
![image](https://user-images.githubusercontent.com/16687385/101986451-3066d400-3c8e-11eb-974f-55c8230ed80f.png)

While the first idea was to simply replace Alkali with Salt this was very quickly discarded because Alkali and Salt are separate items in Sakura and also the fact [Alkali seems to be an important part of making Ramen dough](https://en.wikipedia.org/wiki/Alkaline_pasta)

This issue seeks to discuss a different crafting recipe for Alkali

**Comment by Lgmrszd** [comment]
[This article](https://en.wikipedia.org/wiki/Sodium_carbonate#Production) states that Sodium carbonate is either obtained from trona (mining) or by processing seaweed.

**Reply by LEWeee** [reply]
My suggestion on how to get alkali:
1. Find TFC Seaweed
2. Dry it some way (Sakura adds the Straw web for drying though there is also [this mod](https://www.curseforge.com/minecraft/mc-mods/tfc-drying-rack) which adds a drying rack and allows you to set the number of hours each drying requires)
3. The dried seaweed is put into a firepit. This could result in Alkali itself or it could result in an Alkali Ash (which since there is no item called Alakali Ash would have to be added through ContentTweaker)
4. In case we decide for the latter option the Alkali Ash could then be submerged in fresh water in a TFC Barrel (most likely an Instant recipe) which could result in Alkali itself
5. We could also make it so the barrel recipe results in Alkaline water ([which is how Alkali apparently got into pasta in the first place](https://en.wikipedia.org/wiki/Alkaline_pasta) and could be used in the Ramen Dough recipe since by default Ramen Dough requires both Alkali and Water so with Alkaline Water we would be killing 2 birds with 1 stone, though since there isn't a fluid called Alkaline Water that would likely have to be added through ContentTweaker as well) and then dried in the drying basin from Rustic to get the alkali item itself. This is probably the closest to the process described in the article, but then (to be consistent) we would likely have to add a recipe to turn Salt Water into Salt, which was purposefully not included in TFC by the devs to motivate players to look for Rock Salt, but it was included in Addons like the [Udary Mod](https://terrafirmacraft.com/f/topic/7587-tfc-07929-udary-mod-galenalimonite-extraction-ingame-alloy-calculator-tucker-bag-evaporator-pan-drying-mat/?__cf_chl_jschl_tk__=3c0923592c5d7367dc80e7e3921c6db41c8917f7-1609181099-0-AbBfAhbtEZJyYw20s0umImNFBwSL2YJ5_S73hFcSulh8K7O5hdU7SkWRbDKbfP8UCVdZjcI23UCVZc83O1Q4v6igYX9GWhYLTuk07XWnN7dhZJNMaqaVH2P57EFcV0PtbtVTxbIhVMRU3z-5qGTSvtn01mdUZYFgJZLBxqZHn9C5dKnlSjfuBeegP1T51R_YOqN-u0B_zurD2gmrOgnpqdYdwgYcp-LISIoFLq4BKnegJaYpcUbkJlJKBWuyErI0zb9Z6jauXTKHM5TNtcC__JyxVywksVUMvMUW8KBYj7hYyVG3Zr1VVBf358f_btOOvMa743A-661YaKdrWDvD5rrwTHL6hsvHvgKl461Aig_yCWLOIeAVQN1ekvvaTZhiw61Fp-uPFBd7pKs3oOQb9yH10nA6dvad23IvBcHcUNSSmOCBGoIvjMAI8lm8JGFSkuv7R6_6Os5s1_oZr8TqqN8)).

Also, the only recipe the Rustic Drying Basin has by default is turning Ironberry Juice into piles of Iron Dust, which is a bit overpowered so I would personally suggest just removing the Rustic Drying Basin altogether.

**Comment by LEWeee** [comment]
So since Seaweed doesn't generate in TFC here are some alternatives, at the courtesy of the geology enthusiast:
![obrázok](https://user-images.githubusercontent.com/16687385/103665389-a7347b80-4f73-11eb-979b-0e589c57c092.png)
![obrázok](https://user-images.githubusercontent.com/16687385/103665404-aac80280-4f73-11eb-81dc-e35ef1f5f302.png)
